packages/vehicles/src/utils.js

```diff
@@ -0,0 +1,299 @@
+/* eslint-disable no-unused-vars */
```

Comment:
I think you should remove unused variable declarations and remove this eslint exception.